.github/workflows/smoke-tests.yml

```
@@ -32,10 +32,15 @@ jobs:
           python3 -m venv .venv
           source .venv/bin/activate
           pip install -r requirements.txt
+      - name: Create session ID
+        run: |
+          runner_name=${{ runner.name }}
+          echo "SESSION_ID=${runner_name##*-}" >> "$GITHUB_ENV"
```

Wouldn't that work a little bit more reliably? 
```suggestion
          echo "SESSION_ID=$(( (${runner_name##*-}) % 256 ))" >> "$GITHUB_ENV"
```